lib/features/mailbox/presentation/widgets/mailbox_folder_tile_builder.dart

```diff
@@ -15,7 +15,7 @@ typedef OnOpenMailboxFolderClick = void Function(MailboxNode);
 typedef OnSelectMailboxFolderClick = void Function(MailboxNode);
 typedef OnMenuActionClick = void Function(RelativeRect, MailboxNode);
 typedef OnDragItemAccepted = void Function(List<PresentationEmail>, PresentationMailbox);
-typedef OnLongPressSpamReport = void Function(MailboxNode);
+typedef OnLongPressMailboxFolderClick = void Function(MailboxNode);
```

Comment:
@@ what is this change?

Reply:
This func has a purpose when users of LongPress on mobile will happen 2 cases
1. If it is `SpamMailbox` will show enableSpamReport/disableMailbox
2. If it is Team-Mailbox (`parentId`!= null) and Folder-Mailbox will show `hideMailbox`

So I changed the name for clarity